ci-operator/step-registry/hypershift/kubevirt/baremetalds/odf/hypershift-kubevirt-baremetalds-odf-chain.yaml

```diff
@@ -1,7 +1,8 @@
 chain:
   as: hypershift-kubevirt-baremetalds-odf
   steps:
-  - ref: hypershift-kubevirt-baremetalds-redhat-operators
```

Comment:
Is this expected? Will this have any impact on the existing job?

Reply:
It got introduced with https://github.com/openshift/release/pull/45998
to consume ODF from a custom catalog source named `redhat-operators-v4-14` that statically points to `registry.redhat.io/redhat/redhat-operator-index:v4.14` instead of relying on the standard `redhat-operator` catalog source as available on the cluster for our customers.
I don't see any reason for that and I tend to think it will test something different then what the customers are going to consume so in my opinion we should revert it (eventually fixing bugs in more recent ODF releases if there).